From patchwork Fri Jan 24 02:57:26 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11349571
Return-Path: <SRS0=8och=3N=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id E351517EA
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 24 Jan 2020 02:57:34 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.kernel.org (Postfix) with ESMTP id BD90321D7E
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 24 Jan 2020 02:57:34 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=canb.auug.org.au header.i=@canb.auug.org.au
 header.b="cvtN5YcH"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1730547AbgAXC5b (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Thu, 23 Jan 2020 21:57:31 -0500
Received: from ozlabs.org ([203.11.71.1]:46361 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1729841AbgAXC5b (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 23 Jan 2020 21:57:31 -0500
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 483kMW2mW6z9sRK;
        Fri, 24 Jan 2020 13:57:27 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1579834648;
        bh=q8ffQqdirptsDf8OczDeNcXp3ch/G5iyXyRmUhpQ7xA=;
        h=Date:From:To:Cc:Subject:From;
        b=cvtN5YcHGJtpouxbY2undLOtEunTEaJ3Z9Jkztl/HmNzAQuRUJakQHM8Kc/NPU1Tf
         MPGh3HHHbrCFRQg9DfW/d1XY2xaogzv4EP/xFWIZX/snz7TDo9FpbibHMgEIZK+moK
         FSLCVu+F+82aSlStdqFuI8DpVyvxOB2/PZswHoqhMRPI+foiyuse6tT6ZkrSdqx3wY
         K7yyEN1AwoLZsaxGA17+p+C8no4XExo/qt4K/i2qJZcF01EQ0GJcxwUTKF41uS6oS9
         hkq0Gej50KzTS/sLjATFQI4gFIyETcWgK6I+pbhbzlfyKLG+uFocuaT639LfZCDKGt
         WI6bjTSyD0OGQ==
Date: Fri, 24 Jan 2020 13:57:26 +1100
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>,
 Radim =?utf-8?b?S3LEjW3DocWZ?= <rkrcmar@redhat.com>,
 KVM <kvm@vger.kernel.org>
Cc: Linux Next Mailing List <linux-next@vger.kernel.org>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        "Kirill A. Shutemov" <kirill@shutemov.name>,
        Linus Torvalds <torvalds@linux-foundation.org>,
        Sean Christopherson <sean.j.christopherson@intel.com>,
        Andrew Morton <akpm@linux-foundation.org>
Subject: linux-next: manual merge of the kvm tree with Linus' tree
Message-ID: <20200124135726.5fa8d146@canb.auug.org.au>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  mm/huge_memory.c

between commit:

  97d3d0f9a1cf ("mm/huge_memory.c: thp: fix conflict of above-47bit hint address and PMD alignment")

from Linus' tree and commit:

  0638468d2282 ("mm: thp: KVM: Explicitly check for THP when populating secondary MMU")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc mm/huge_memory.c
index a88093213674,9b3ee79d0edf..000000000000
--- a/mm/huge_memory.c
